src/main/java/org/apache/maven/buildcache/checksum/MavenProjectInput.java

```diff
@@ -180,9 +182,21 @@ public MavenProjectInput( MavenProject project,
         {
             if ( propertyName.startsWith( CACHE_EXCLUDE_NAME ) )
             {
+                String propertyValue = properties.getProperty( propertyName );
+                Path path = Paths.get( propertyValue );
+                filteredOutPaths.add( path );
+                if ( LOGGER.isDebugEnabled() )
+                {
+                    LOGGER.debug( "Adding an excludePath from property '{}', values is '{}', path is '{}' ",
+                            propertyName, propertyValue, path );
+                }
+
                 filteredOutPaths.add( Paths.get( properties.getProperty( propertyName ) ) );
             }
         }
+        CacheUtils.debugPrintCollection( LOGGER, filteredOutPaths,
+                "List of excluded paths (checked either by fileName or by startsWith prefix)",
+                "Pah entry" );
```

Comment:
'Pah' entry is a medical term likely :-)

Reply:
@AlexanderAshitkin fixed